cpp/cmake/thirdparty/get_arrow.cmake

```diff
@@ -62,10 +62,26 @@ function(find_libarrow_in_python_wheel PYARROW_VERSION)
   # to builds of GTest. Similarly, benchmarks will not work without updating GBench (and possibly
```

Comment:
Can we update the comment above this? It states "pyarrow's `libarrow.so` is compiled for manylinux2014" which is now only true for a particular build.

Reply:
Yup.